### PR TITLE
Bookstore: styling

### PR DESCRIPTION
**Pull request by Dorcas126**
Now that all the functionality is done, I have to add styling.

# Project description

- Add any missing JSX elements to the project (without adding extra functionality)
- The final design should match Zeplin's design (this includes all elements + the same styling)
- Use the CSS rules detailed in Zeplin to style the website.

### CODE REVIEW

File: src/style/Header.module.css [hunk 1]
```diff
@@ -1,19 +1,58 @@
...
+.list::-webkit-scrollbar {
+  display: none;
+}
+
+.loading {
+  height: 500px;
+  display: flex;
+  gap: 0.8rem;
+  align-items: center;
+  justify-content: center;
```
Comment (thecodechaser) [inline src/style/Header.module.css]: - [ ] Please make sure to increase the left margin for the progress bar, It should be near the `UPDATE PROGRESS` button as on the design.

### Your design

image

### Zepline design

image

File: src/style/Header.module.css [hunk 2]
```diff
@@ -1,19 +1,58 @@
...
+  overflow-y: scroll;
+}
+
+/* Books.js styling */
+.list::-webkit-scrollbar {
+  display: none;
+}
+
+.loading {
+  height: 500px;
```
Comment (thecodechaser) [inline src/style/Header.module.css]: - [ ] Please make sure the add book section starts on the same line vertically as the book container so please add left margins to it. Please also fix the categories and add book buttons it should match the Zeplin design.

### Your design

image

### Zeplin design

image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @Dorcas126👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- Nice UI design. ✔️
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/routes/Home.js [hunk 3]
```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import Books from '../components/Books';
 import AddBooks from '../components/AddBooks';
+import '../index.css';
 
 const Home = () => (
```
Comment (TedLivist) [inline src/routes/Home.js:6]: - [ ] You have still not implemented the change from the previous reviewer. Kindly compare the designs and stick to it as much as possible

  - The percent score should move closer to the right, next to the Update Progress button
  - The Book form should be made wider to cover the entire length of the container just like the container of the book.

  Your Implementation | Zeplin Design
  :---:|:---:
  image | image

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @Dorcas126,

✔️ Font style follows the design

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Book.js [hunk 4]
```diff
@@ -1,30 +1,63 @@
...
+          <div className={styles.progress_content}>
+            <span className={styles.progress_text}>
+              {progressText}
+              %
+            </span>
+            <span className={styles.progress_complete}>Completed</span>
+          </div>
+        </div>
+      </main>
+      <main className={styles.current}>
```
Comment (souad988) [inline src/components/Book.js]: [OPTIONAL] main tag is used to wrap the different sections of a page exclud only the header and footer or side navigation bar

File: src/style/Header.module.css [hunk 5]
```diff
@@ -1,19 +1,59 @@
...
 }
 
-.row_1 {
+.lesson_content {
   display: flex;
   align-items: center;
-  justify-content: space-between;
+  gap: 7rem;
+  border-right: 3px solid #e8e8e8;
+  padding-right: 90px;
```
Comment (souad988) [inline src/style/Header.module.css]: Using a fixed right padding  to align the items is not a good idea , I suggest that you wrap the progress bar and the update progress section in on container and set justify content of their parent wrapper to space-between to get the desired output like the design as mentioned by the previous reviewers 

Capture d’écran du 2023-07-28 13-54-55

if you need further help don't hesitat to ask

**Review by souad988 — CHANGES_REQUESTED** [review]
Hi @Dorcas126 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
Good file structure
Create multiple components for reusability 
Overall work is good

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mikethreels — APPROVED** [review]
Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Good job following the instructions of the previous CR.
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**